### ISSUE TITLE
NVDA And Outlook 2016 Autofill

### ISSUE THREAD

**Issue by Geoff236** [nvaccess/nvda#10649]
Good afternoon. I am currently using the latest stable version of NVDA on a Windows 10 computer and have Office 2016 installed. My issue in Microsoft Outlook is the auto fill when I go to send an e-mail message. I'll type the first letter of the name of the person I'm addressing the message to, and NVDA doesn't read the rest of the name back to me. This has been going on for quite some time. Maybe this issue has already been reported to you but I thought I'd send it just in case it hasn't.
Any help on this matter would be appreciated.
GeoffYou have arrived at a default issue template (bug report).
It is preferable to [choose the type of issue](https://github.com/nvaccess/nvda/issues/new/choose) to submit.
Hint: Swap to the preview tab to be able to click the links.
Direct links to the templates:
- [Bug report](https://github.com/nvaccess/nvda/issues/new?template=bug_report.md)
- [Feature request](https://github.com/nvaccess/nvda/issues/new?template=feature_request.md)

**Comment by Adriani90** [comment]
Thanks for reporting. This is a duplicate of #8502. Thus, closing as duplicate.
